requirements.txt

```diff
@@ -1,8 +1,8 @@
-cirq-core>=0.12.0
-cirq-google>=0.12.0
+cirq-core>=0.14.0
+cirq-google>=0.14.0
```

Comment:
actually https://github.com/quantumlib/ReCirq/issues/247 has a bit more context.

The "previous" check is currently disabled because I decreed it _was_ an undue burden, so maybe we can try un-disabling the `previous` check

Reply:
We would like to use the new Transformers API and start upgrading everything to support the new APIs as a good test in order to smoke test everything in preparation of Cirq 1.0.  Supporting cirq 0.13 would prevent that I think.